pkgs/servers/matrix-synapse/default.nix

```diff
@@ -1,8 +1,20 @@
-{ lib, stdenv, python3
+{ lib, stdenv, fetchFromGitHub, python3, openssl
 , enableSystemd ? stdenv.isLinux
 }:
 
-with python3.pkgs;
+with python3.pkgs.override {
+  overrides = self: super: {
+    service-identity = super.service-identity.overrideAttrs (oldAttrs: rec {
+      version = "18.1.0";
```

Comment:
Could you add a comment line to the `service-identity` package saying its version was duplicated here, in case a security vulnerability arises over there so that people know to upgrade here too.

Reply:
Amended.